app/models/framework_response/collection.rb

```diff
@@ -0,0 +1,51 @@
+class FrameworkResponse
+  class Collection < FrameworkResponse
+    validates :value_text, absence: true
+    validate :validate_presence, on: :update, if: -> { framework_question.required && parent.nil? }
+    validate :validate_details_collection, on: :update, if: -> { response_details? }
+
+    def self.sti_name
+      'collection'
+    end
+
+    def value
+      value_json.presence || []
+    end
+
+    def value=(answer)
```

Comment:
Feels weird mixing response/answer concepts, here. What about killing the word answer in this new implementation are just using response?

Reply:
```ruby
    def value=(response_value)
```

Reply:
I've gone with `raw_value`, let me know your thoughts